jupyter-extension/cypress/integration/mount.ts

```diff
@@ -18,7 +18,9 @@ describe('mount', () => {
     cy.findAllByText('/ pfs').should('have.length', 2);
     cy.findAllByText('Load').first().click();
     cy.findAllByText('Unload').should('have.length', 1);
-    cy.findAllByText('default_images').first().click();
+    cy.get('#jupyterlab-pachyderm-browser-pfs')
```

Comment:
Not exactly sure why this change is needed now, but the problem with searching `findByText` without an id is that `default_images` also appears as text in the repo list. When I debugged this locally the text is the element that was clicked which obviously does nothing. Changing this to findByText in the FileBrowser ID scope fixes this issue.